wiki/Guides/Videos_from_YouTube/en.md

```diff
@@ -0,0 +1,29 @@
+---
+tags:
+  - how to download videos from YouTube
+  - how to use YouTube videos
+---
+
+# Videos from YouTube
+
+[YouTube](https://youtube.com) is a very convenient resource. If a song has a music video, it will most likely be on YouTube, and probably compressed at that golden point where the video looks decent but will not be a pain to download.
+
+This tutorial is intended for those who **can not** find the video elsewhere but YouTube.
+
+## Using online tools
+
+[ytmp3.cc](https://ytmp3.cc) is an online YouTube converter service. Using ytmp3.cc is very straightforward. Paste the YouTube video link into the text box, select "mp4", and click "Convert". After conversion is finished, click "Download" to download the video file.
+
+## Using youtube-dl
+
+[youtube-dl](https://youtube-dl.org) is a command-line program to download videos from YouTube and other sites. The fastest way to download from YouTube using youtube-dl is by using the `-f bestvideo` option:
+
+```cmd
+youtube-dl.exe -f bestvideo <video link>
```

Comment:
I think that for these who badly want to use it, but don't quite know where to past the command, you can mention the terminal, or the console, or PowerShell.

Reply:
Am willing to provide screenshots for this if needed